### PR TITLE
Remove extra `<template>` tags from components

### LINKED ISSUE #11910: CIS Benchmark: Scan Create form shows empty content
## Steps to reproduce

- Install CIS Benchmark cluster tool helm chart
- Cluster --> Tools --> install CIS Benchmark --> refresh
- Navigate to resource menu `CIS Benchmark` --> Scans --> Create

## Actual behavior

No content renders on page. No errors or warnings in log.

## Expected behavior

image

### LINKED ISSUE #11912: CIS Benchmark: No content shown in profile detail page
## Steps to reproduce

- Install CIS Benchmark cluster tool helm chart
- Cluster --> Tools --> install CIS Benchmark --> refresh
- Navigate to resource menu CIS Benchmark --> Profiles --> click on a scan

## Actual behavior

No content renders on page. No errors or warnings in log.

## Expected behavior

Content is rendered in page

### LINKED ISSUE #11913: CIS Benchmark: No content shown in profile create/edit page
## Steps to reproduce

- Install CIS Benchmark cluster tool helm chart
- Cluster --> Tools --> install CIS Benchmark --> refresh
- Navigate to resource menu CIS Benchmark --> Profiles --> create

## Actual behavior

No content renders on page. No errors or warnings in log.

## Expected behavior

Content is rendered in page

### PR DESCRIPTION
**Pull request by rak-phillip**
### Summary
This resolves a breaking change in Vue3 that treats `<template>` tags with no special directives as plain elements[^1] by removing the extra `<template>` tags from each component.

> `<template>` tags with no special directives (v-if/else-if/else, v-for, or v-slot) are now treated as plain elements and will result in a native `<template>` element instead of rendering its inner content.

This PR addresses more than just the CIS Benchmark, but I decided to address all of these instances after identifying the root cause to be the same for each occurrence 

Fixes #11910
Fixes #11913
Fixes #11912

[^1]: https://v3-migration.vuejs.org/breaking-changes/#other-minor-changes

### Occurred changes and/or fixed issues

- remove extra `<template>` tags from components

### Technical notes summary

It might be a good idea to replace these instances of `<template>` with `<div>` tags instead.. We might have to evaluate each instance individually.

I hacked together the following for detecting files that contain more than one instance of `<template>`

```
➤ grep -rc --exclude-dir=node_modules/ '<template>' . | grep -v ":0" | grep -v ":1"
./shell/components/TabTitle.vue:2
./shell/components/form/Labels.vue:2
./shell/components/formatter/InternalExternalIP.vue:2
./shell/detail/helm.cattle.io.projecthelmchart.vue:2
./shell/edit/auth/ldap/config.vue:2
./shell/edit/provisioning.cattle.io.cluster/tabs/registries/index.vue:2
./shell/edit/cis.cattle.io.clusterscan.vue:2
./shell/edit/cis.cattle.io.clusterscanprofile.vue:2
./shell/edit/resources.cattle.io.backup.vue:2
./shell/edit/resources.cattle.io.restore.vue:2
./shell/pages/c/_cluster/uiplugins/AddExtensionRepos.vue:2
./shell/promptRemove/management.cattle.io.roletemplate.vue:2
```

### Areas or cases that should be tested

It would be best to confirm that each component changed does not regress in any way.

### Areas which could experience regressions

Each component that has been change could possibly regress as a result of this change.

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

File: shell/components/formatter/InternalExternalIP.vue [hunk 1]
```diff
@@ -38,21 +38,19 @@ export default {
       -
     </template>
     /
-    <template>
```
Comment (richard-cox) [inline shell/components/formatter/InternalExternalIP.vue:41]: took a quick look, i think this fixes https://github.com/rancher/dashboard/issues/11819? cc @mantis-toboggan-md

File: shell/edit/auth/ldap/config.vue [hunk 2]
```diff
@@ -95,305 +95,303 @@ export default {
 
 <template>
   <div @update:value="$emit('update:value', model)">
-    <template>
```
Comment (richard-cox) [inline shell/edit/auth/ldap/config.vue:98]: not sure if it's related to this change, but can't validate this change due to it, when i go to any of the ldap auth providers the page fails to render. console error

image

**Review by richard-cox — CHANGES_REQUESTED** [review]
- This should definitely be called out on an extension upgrade guide. It'll be nestled in general vue3 migration guides, but we've hit it often enough for it likely to be an issue for them too. Are we tracking this stuff anywhere?

File: shell/edit/auth/ldap/config.vue (hunk 2 above)
Comment (rak-phillip) [reply shell/edit/auth/ldap/config.vue:98]: Thanks for the heads up. I can investigate and create another issue if one is required.

Comment (rak-phillip) [reply shell/edit/auth/ldap/config.vue:98]: We get this error in `master` without this change present and I've traced the issue down to a circular reference in our components.

We import `saml.vue` in `config.vue`:

https://github.com/rancher/dashboard/blob/14089aba62240aa4f7af9eb27511e343e2f352e0/shell/edit/auth/ldap/config.vue#L8

And in `saml.vue`, we import `config.vue`:

https://github.com/rancher/dashboard/blob/14089aba62240aa4f7af9eb27511e343e2f352e0/shell/edit/auth/saml.vue#L12

I think that fixing this issue will be outside of the scope of this PR, so I'm working on #11934 to track the issue.

**Comment by rak-phillip** [comment]
> * This should definitely be called out on an extension upgrade guide. It'll be nestled in general vue3 migration guides, but we've hit it often enough for it likely to be an issue for them too. Are we tracking this stuff anywhere?

I added a section to #10658 so that we don't lose track of this.

File: shell/components/formatter/InternalExternalIP.vue (hunk 1 above)
Comment (richard-cox) [reply shell/components/formatter/InternalExternalIP.vue:41]: Will leave any follow up for that PR

**Review by richard-cox — APPROVED** [review]

**Pull request merged**